db/migrate/20230904121544_create_journal_entries.rb

```diff
@@ -0,0 +1,16 @@
+class CreateJournalEntries < ActiveRecord::Migration[7.0]
+  def up
+    create_table :journal_entries do |t|
+      t.integer :journal_id, :null => false, index: true, foreign_key: true
```

Comment:
`:null => false` that's an old format. Let's use the newer formats, `null: false`